### PR TITLE
feat: BasicCrawler.export_data helper

### LINKED ISSUE #211: `BasicCrawler.export_to` should redefine the `key` and `content_type` parameters
- the `key` parameter is mandatory, but the user has to browse the source code or wait for a crash to know that
- going blind, I tried passing `foo.csv` to `key` and got a file called `foo.csv.json`, which we may also want to handle somehow

### PR DESCRIPTION
**Pull request by janbuchar**
- closes #211

### CODE REVIEW

File: src/crawlee/storages/dataset.py [hunk 1]
```diff
@@ -189,6 +190,32 @@ async def get_data(self, **kwargs: Unpack[GetDataKwargs]) -> DatasetItemsListPag
         # https://github.com/apify/apify-sdk-python/issues/140
         return await self._resource_client.list_items(**kwargs)
 
+    async def write_to(self, content_type: Literal['json', 'csv'], destination: TextIO) -> None:
```
Comment (vdusek) [inline src/crawlee/storages/dataset.py:193]: Is `write_to` named after the same method in Crawlee JS? Why not use `export_data` as on the crawler?

**Review by vdusek — CHANGES_REQUESTED** [review]
LGTM, just naming and one more thing: the class `ExportToKwargs` - couldn't it be removed now?

File: src/crawlee/storages/dataset.py (hunk 1 above)
Comment (janbuchar) [reply src/crawlee/storages/dataset.py:193]: No, that's mine. I imagine having `export_data` *and* `export_to` would be confusing. If you have a better name in mind, I'm open to suggestions.

**Comment by janbuchar** [comment]
> the class `ExportToKwargs` - couldn't it be removed now?

No, it's still used in `Dataset.export_to`.

**Review by vdusek — APPROVED** [review]

**Pull request merged**